.github/workflows/ci.yml

```diff
@@ -96,7 +96,11 @@ jobs:
       - uses: software-mansion/setup-scarb@v1.3.2
       - uses: software-mansion/setup-universal-sierra-compiler@v1
       - name: Run tests
-        run: cargo test --release -p sncast
+        env:
+          NODE_URL: ${{ secrets.NODE_URL }}
+        run: |
+          export FORK_NETWORK_URL="${NODE_URL}:7070/rpc/v0_7"
```

Comment:
If we construct the exact URL in the code, it will enforce the use of our node URL, which could be problematic, especially for external contributors. We might consider appending `/rpc/v0_x` at the code level, but only if this path is common across all node providers.